api/controllers/UserController.js

```diff
@@ -276,13 +276,13 @@ module.exports = {
     Network.find(req.param('networkId'))
     .then(network => Community.query().where('network_id', network.id).select('id'))
     .then(rows => _.map(rows, 'id'))
-    .then(ids => Search.forUsers({
-      communities: ids,
+    .then(communityIds => Search.forUsers({
+      communities: communityIds,
       limit: req.param('limit') || 20,
       offset: req.param('offset') || 0
-    }).fetchAll({withRelated: ['skills', 'organizations']}))
-    .tap(users => total = (users.length > 0 ? users.first().get('total') : 0))
-    .then(users => users.map(u => UserPresenter.presentForList(u)))
+    }).fetchAll({withRelated: ['skills', 'organizations', 'memberships']})
+      .tap(users => total = (users.length > 0 ? users.first().get('total') : 0))
```

Comment:
the `tap` and `then` are actually inside the block that starts `communityIds => `
Made a different change, see if it looks right to you now